dpgen/simplify/simplify.py

```diff
@@ -315,14 +315,16 @@ def make_fp_configs(iter_index, jdata):
     create_path(work_path)
     picked_data_path = os.path.join(iter_name, model_devi_name, picked_data_name)
     systems = get_multi_system(picked_data_path, jdata)
+    ii = 0
     jj = 0
     for system in systems:
         for subsys in system:
-            task_name = "task." + fp_task_fmt % (0, jj)
+            task_name = "task." + fp_task_fmt % (ii, jj)
```

Comment:
Instead, I suggest using `MultiSystems` to `append` systems, like what dpgen2 does, so there will be no errors to add different systems.
https://github.com/deepmodeling/dpgen2/blob/008d1dfe91f3628a7039dca5d3af10c1b4bca506/dpgen2/op/collect_data.py#L67-L73